src/Symfony/Component/Intl/DateFormatter/IntlDateFormatter.php

```diff
@@ -143,15 +143,15 @@ class IntlDateFormatter
      * @see http://userguide.icu-project.org/formatparse/datetime
      *
      * @throws MethodArgumentValueNotImplementedException When $locale different than "en" or null is passed
-     * @throws MethodArgumentValueNotImplementedException When $calendar different than GREGORIAN is passed
+     * @throws MethodArgumentValueNotImplementedException When $calendar different than GREGORIAN is passed or null is passed
```

Comment:
This change is wrong (no exception will be thrown when null is passed).
The docblock that needs to be changed is the `@param` above.